.github/workflows/validate.yml

```diff
@@ -160,3 +160,29 @@ jobs:
           cache: maven
       - name: Maven build
         run: etc/scripts/build.sh
+  build-latest-java:
+    needs: prime-build
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        os: [ ubuntu-22.04 ]
+    runs-on: ${{ matrix.os }}
```

Comment:
If the matrix contains only `ubuntu-22.04`, can this value be inlined with `runs-on` ?